.github/workflows/dev-test.yml

```diff
@@ -0,0 +1,140 @@
+# Run tests against developer push
+
+# Runs jobs on linux and windows to maintain compatibility.
+# The main repo is checked out into a sub-directory because an optional
+# second repo may be checked out for testing.
+
+name: dev-test
+
+on:
+  push:
```

Comment:
Wouldn't we want this to run also on PRs?

Reply:
I was initially setting up for ensuring developer pushes got tested on both windows and linux (I am still a windows cmd user but the azure pipelines are all linux).  If you want to switch the PR check build to here that would be great.  In that case, it should probably be renamed.  I am open to suggestions (ci-test.yml?).

Reply:
`pr-test.yml`?